Celeste.Mod.mm/Mod/Everest/Everest.Loader.cs

```diff
@@ -295,11 +319,22 @@ public static class Loader {
                             continue;
                         }
                     }
+                    if(zip.ContainsEntry(".everestignore")) {
```

Comment:
Why not do this in the preceding loop over the zip entries? `ContainsEntry` presumably does another loop, which seems wasteful. Also very cursed thought, should we support people .everestignoring any of the yaml files that are checked for above?